public/components/common/hooks/use-value-suggestion.ts

```diff
@@ -39,19 +40,29 @@ export const useValueSuggestion = (
   const [isLoading, setIsLoading] = useState(true);
   const data = getDataPlugin();
   const indexPattern = useIndexPattern();
-  //const { filters } = useFilterManager();
 
   const getOptions = (): string[] => {
     return options?.filter((element) => element.toLowerCase().includes(query.toLowerCase())) || [];
   };
 
   const getValueSuggestions = async (field) => {
+    const boolFilter =
+      filterDrillDownValue && filterDrillDownValue.value !== ""
+        ? [
+            {
+              term: {
+                [filterDrillDownValue.field]: `${filterDrillDownValue.value}`,
+              },
+            },
+          ]
+        : [];
```

Comment:
If the previous comment was followed, this could be replaced by
```
const boolFilter = boolFilters.map(filter => ( 
    {
        term: {
            [filterDrillDownValue.field]:  filterDrillDownValue.value
        }
    } ) );
```
because we would always know boolFilters is, at least, an empty array. And if it isn't, it will have a key value pair.